### DISCUSSION TITLE
JK DZ11

### DISCUSSION THREAD

**Discussion by RichGorrod** [Louisvdw/dbus-serialbattery#1090] [General]
Hello,
I have added a DZ11 balancer to a DIY seplos 16s 280aH battery pack. 
There is a Can/ RS485 connection on these. balancers. -
Is it be possible to read this on the cerbo GX / node red.

Any pointers would be greatly appreciated.

Below in the serial data for this unit.

[JK-B1A24S-RS485-protocol-english (1).pdf](https://github.com/user-attachments/files/16053583/JK-B1A24S-RS485-protocol-english.1.pdf)

.Many thanks.

Rich

**Comment by mr-manuel** [comment]
If I remember correctly there was already someone else who connected successfully the balancer. In that case you get only the cell voltages and if it's balancing or not. The port description is like on other JKBMS wrong. You find a lot of informations about that in the closed issues.

**Comment by RichGorrod** [comment]
Many thanks Manuel,

Is there any chance you could point me in the right direction.

I will have a look now myself too.

Many thanks again.

Cheers
Rich

**Reply by mr-manuel** [reply]
Just connect the balancer properly and it should work. You only have to connect it the right way: https://github.com/Louisvdw/dbus-serialbattery/discussions/250

**Comment by RichGorrod** [comment]
Hello,

Sorry for the message again.

I have JK BMS can have connected the JK RS485 adaptor and it show up instantly.

When connecting to the balancer there is no response.

It does not show up on the JK BMS windows software.

When connected with bluetooth to my phone there are no options to change the protocol just the devices address.

I am i missing something.

![image](https://github.com/Louisvdw/dbus-serialbattery/assets/141816621/aa110ce1-8814-4ef1-8075-c8b373b25a06)

many thanks for all your help.

Rich

**Reply by mr-manuel** [reply]
It won't work, if you do not have connected at least 4 cells.

**Reply by RichGorrod** [reply]
sorry this is just a picture of the unit and the RS485 / CAN have 16 cells connected and all show up in the bluetooth app.
![JK App 1](https://github.com/Louisvdw/dbus-serialbattery/assets/141816621/b9fbdf42-c6ce-4f02-84d7-e9164e7361a4)
![JK App 2](https://github.com/Louisvdw/dbus-serialbattery/assets/141816621/dd12c483-36ee-43ca-86a8-b0d0752aa089)

Will do some more investigation this evening.

Cheers

Rich
